save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -46,9 +47,12 @@ class WarnPlugin(
         val warnPluginConfig = testConfig.pluginConfigs.filterIsInstance<WarnPluginConfig>().single()
         val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().singleOrNull()
 
-        return files.chunked(warnPluginConfig.batchSize ?: 1).map { chunk ->
+        val listTestResult: MutableList<TestResult> = mutableListOf()
```

Comment:
If diktat complains on early declaration here, than it's a bug in diktat, because we should be correctly detecting such cases. Please check if it is reproducible and open an issue in that case

Reply:
https://github.com/cqfn/diKTat/issues/934